src/index.json

```diff
@@ -201,6 +201,61 @@
                 "sha256Digest": "81b9a7f1824bffcbd6e72680891476151dde868e030cdbf1bfd9135e1a8f3447"
             }
         ],
+        "appconfig": [
```

Comment:
Why is this change here?

Reply:
That's the result of using `azdev extension update-index`